lib/ansible/module_utils/iosxr.py

```diff
@@ -40,10 +73,14 @@
     'password': dict(fallback=(env_fallback, ['ANSIBLE_NET_PASSWORD']), no_log=True),
     'ssh_keyfile': dict(fallback=(env_fallback, ['ANSIBLE_NET_SSH_KEYFILE']), type='path'),
     'timeout': dict(type='int'),
+    'authorize': dict(fallback=(env_fallback, ['ANSIBLE_NET_AUTHORIZE']), type='bool'),
```

Comment:
Is this right, I didn't think iosxr had privilege mode.

Reply:
right, it doesn't. will remove it.